### PR TITLE
add stellar.expert and stellarchain.io on tx submission success box

### PR DESCRIPTION
**Pull request by jeesunikim**
tx-success

links used for:
**Testnet**
- stellar.expert: https://stellar.expert/explorer/testnet/tx/3320ead9a67b2cc66376313b33f12c108ac3eee5e72076422470b70da44c7642 
- stellarchain.io: https://testnet.stellarchain.io/transactions/3320ead9a67b2cc66376313b33f12c108ac3eee5e72076422470b70da44c7642

**Mainnet**
- stellar.expert: https://stellar.expert/explorer/public/tx/3fad66fbc0ad96d536bdd79a5358740bda26a8ef04a3b4ebfff04041642c473f
- stellarchain.io: https://stellarchain.io/transactions/3fad66fbc0ad96d536bdd79a5358740bda26a8ef04a3b4ebfff04041642c473f

### CODE REVIEW

File: src/app/(sidebar)/transaction/submit/page.tsx [hunk 1]
```diff
@@ -220,6 +225,36 @@ export default function SubmitTransaction() {
...
+          note={<></>}
+          footerLeftEl={
+            IS_BLOCK_EXPLORER_ENABLED ? (
+              <>
+                <Button
+                  size="md"
+                  variant="tertiary"
+                  onClick={() => {
+                    openUrl(
+                      `https://stellar.expert/explorer/${network.id}/tx/${submitRpcResponse.hash}`,
```
Comment (quietbits) [inline src/app/(sidebar)/transaction/submit/page.tsx:237]: It looks like mainnet on StellarExpert is `public` ([example](https://stellar.expert/explorer/public/tx/876f9b8a791ef20e9cd4ef9e03d3082f51e1f2db1661b70cb0885ddf946fa771)).

File: src/app/(sidebar)/transaction/submit/page.tsx [hunk 2]
```diff
@@ -220,6 +225,36 @@ export default function SubmitTransaction() {
...
+                >
+                  View on stellar.expert
+                </Button>
+
+                <Button
+                  size="md"
+                  variant="tertiary"
+                  onClick={() => {
+                    openUrl(
+                      `https://${network.id}.stellarchain.io/transactions/${submitRpcResponse.hash}`,
```
Comment (quietbits) [inline src/app/(sidebar)/transaction/submit/page.tsx:249]: Mainnet URL on StellarChain doesn't include network ([example](https://stellarchain.io/transactions/876f9b8a791ef20e9cd4ef9e03d3082f51e1f2db1661b70cb0885ddf946fa771))

**Review by quietbits — COMMENTED** [review]
We'll need to create a helper function to get the correct StellarExpert and StellarChain URL for the mainnet because they don't seem to follow the same pattern as testnet.

File: src/app/(sidebar)/transaction/submit/page.tsx (hunk 1 above)
Comment (jeesunikim) [reply src/app/(sidebar)/transaction/submit/page.tsx]: thank you! updated it and tested on mainnet

**Pull request merged**